cumulusci/tasks/release_notes/task.py

```diff
@@ -62,25 +66,29 @@ def _run_task(self):
             "prefix_beta": self.project_config.project__git__prefix_beta,
             "prefix_prod": self.project_config.project__git__prefix_release,
         }
+        for repo in self.options["repos"]:
+            print(f"Repo: {repo}\n")
+            github_info["github_repo"] = repo
+            generator = GithubReleaseNotesGenerator(
+                self.github,
+                github_info,
+                self.project_config.project__git__release_notes__parsers.values(),
```

Comment:
This is where you'd have to intervene to make this viable for projects that customize their release notes. Each loop iteration would have to source the `cumulusci.yml` for the remote repository and parse out its relevant configuration here.

Reply:
@davidmreed I'm struggling to find an easy solution to this. Have we had this issue in the past?

Reply:
Check out `cumulusci.core.dependencies.github.get_remote_project_config()`